third-party/imgui/realsense_imgui.cpp

```diff
@@ -0,0 +1,486 @@
+/* License: Apache 2.0. See LICENSE file in root directory. */
+/* Copyright(c) 2024 Intel Corporation. All Rights Reserved. */
+
+//Overloading imgui functions to accommodate realsense GUI
```

Comment:
So we must include this header before the IMGUI header?

Reply:
I'm not sure what you meant, we include this header if we use one of the helper\overloading functions, we do use it almost every time we use imgui.